assists/cdo.py

```diff
@@ -1,4 +1,4 @@
-#/*
+# /*
```

Comment:
Was the formatting / cleanup done via a linter/pep8/other utility ?

If so, we need to document that in the commit header, so we can ensure consistency in the future.

Reply:
okay, sure will document the utility (or format) in the commit header.

I used "yapf" (https://github.com/google/yapf) - by default it does pep8 formatting, which is what I used. It supports other formats however.

Reply:
Sounds good. The plan was to do pep8, so that is the preference (for now).